internal/exec/terraform_test.go

```
@@ -6,6 +6,7 @@
 	"strings"
 	"testing"
 
+	"github.com/charmbracelet/log"
```

golangci-lint (1.64.8)

[error] import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)